package/MDAnalysis/analysis/hbonds/hbond_autocorrel.py

```diff
@@ -32,7 +32,8 @@
 then the lifetime of these bonds is monitored over time.  Multiple passes
 through the trajectory are used to build an average of the behaviour.
 
-    :math:`C_x(t) = \\left \\langle \\frac{h_{ij}(t_0) h_{ij}(t_0 + t)}{h_{ij}(t_0)^2} \\right\\rangle`
+.. math::
```

Comment:
fyi: If you don't want double-backslashes, make the whole doc string a raw string.